truffle.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   networks: {
-    development: {
+    sample: {
```

Comment:
why this change?

Reply:
because truffle test have happend to error.

truffle v4.0.1

Reply:
and how does this solve that? I've never seen this `sample` network before

Reply:
because truffle 4.0 use built-in blockchain. 
https://github.com/trufflesuite/truffle/releases/tag/v4.0.0
>Caveat: For backwards compatibility, if you have an existing Truffle project with a development or test network specified in your truffle.js file, Truffle will use those networks instead of this new built-in blockchain. To take full advantage of this feature, remove or rename the development and test networks from your truffle.js file. If you're just using the TestRPC for development, consider using the new truffle develop instead!